logging.js

```diff
@@ -192,12 +199,32 @@ export class LoggingClient {
 	}
 
 	legacyErrorBatch(legacyErrors) {
-		const logs = legacyErrors.map(({ message, source, lineno, colno, error, developerMessage }) => new LogBuilder(this._appId)
+		const remainingLegacyErrors = this._throttle(legacyErrors);
+
+		const logs = remainingLegacyErrors.map(({ message, source, lineno, colno, error, developerMessage }) => new LogBuilder(this._appId)
 			.withLegacyError(message, source, lineno, colno)
 			.withError(error)
 			.withMessage(developerMessage)
 			.withLocation()
 			.build());
 		this._logger.logBatch(logs);
 	}
+
+	_throttle(logs) {
+		if (!this._shouldThrottle) {
+			return logs;
+		}
+
+		const now = new Date().getTime();
```

Comment:
I figured we were ok losing a bit of precision here instead of asking for the date again for every loop through the array of logs.

Reply:
Agreed!